src/FinishedList.js

```diff
@@ -0,0 +1,49 @@
+
+
+function FinishedList({location, initialState, onClickTodoItem, onDeleteTodoItem}) {
+
+    this.state = {...initialState};
+
+    this.setState = (nextState) => {
+        this.state = {
+            ...this.state,
+            ...nextState
+        }
+
+        this.render();
+    }
+
+    this.render = (finishedTodo) => {
+
+        const p = document.querySelector('.finished-todo-list-count');
+        p.textContent = finishedTodo.length;
+
+        location.innerHTML = '';
+
+        finishedTodo.map(todo => {
+            const li = document.createElement('li');
+            const btn = document.createElement('button');
```

Comment:
여기도,,, 버튼이 정확히 어떤 버튼인지 명시해주면 아래줄에 클래스 이름을 부여하는 라인까지 내려가지 않아도 이 버튼의 용도를 알 수 있겠죠?
```suggestion
            const singleTodoItem = document.createElement('li');
            const deleteBtn = document.createElement('button');
```